index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>The HTML5 Herald</title>
+    <meta name="description" content="The HTML5 Herald" />
+    <meta name="author" content="SitePoint" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css" />
```

Comment:
- [ ]  It will be nice to add "./" before the file to be linked, thus you will apply the best standard industry practices. Thus you can navigate from the root to the desired files by letting HTML know exactly where to start.